### DISCUSSION TITLE
UI/UX: Increase size of annotation hover hitbox, fix positioning

### DISCUSSION THREAD

**Discussion by Kangaroux** [grafana/grafana#45706] [Dashboards, Panels & Visualizations]
Hovering over an annotation can be difficult since the element is only 16x8 pixels. The hitbox is also positioned incorrectly:

![image](https://user-images.githubusercontent.com/2302541/155013106-8a4209fd-a4ec-40d0-8ff4-ad6695fbd973.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
